### ISSUE TITLE
Patcher.exe crashes on Windows 7

### ISSUE THREAD

**Issue by kalissar30** [Jonesey13/TF-8-Player#1]
Hello.
I tried following your steps but the patcher crashed :(
It still produced a PatchedTowerFall.exe though, not TowerFall8Player.exe
but the produced .exe just seems like a copy of TowerFall.exe

Unhandled Exception : System.InvalidOperationException: Sequence contains no matching element.
   at System.Linq.Enumerable.Single[TSource](IEnumerable`1 source, Func`2 predicate)
   at Patcher.MyTFCommands.PatchModule(ModuleDefinition baseModule)
   at Patcher.Patcher.Patch(String modModulePath)
   at Patcher.Patcher.Main(String[] args)

Config :
Steam installation
Towerfall directory in D:\SteamAdd\SteamApps\common\TowerFall

I copied the content of the Pre Built directory here and replaced the files that already existed.
Hope it's enough to debug !

Cheers

**Comment by Jonesey13** [comment]
Yes the steam version of the game has been updated which breaks the current version of the mod. Atm it should work fine on the humble bundle version.

The IL code on the steam and humble versions have diverged considerably. My guess is that the steam version's source code hasn't really changed much compared to the humble version, but it's been built with a different version of the C# compiler. 

The bottom line is that it is fixable, but I want to fix it in a way that it won't break in the future and that requires a bit of work.

Thanks for reporting the bug :)

**Comment by jarommadsen** [comment]
Any news on work being done on this? I know Matt was working on an 8-player version for GDC but he's been pretty quiet ever since. Only have the Steam version so I lost the 8-player capability and would love to get it back.

**Comment by Jonesey13** [comment]
I have pushed an update which should make the mod work with the steam version again. If you get another error please tell me and I'll try to fix it asap.

**Comment by jarommadsen** [comment]
Thanks so much for the quick response! Everything is back in order!

**Comment by Jonesey13** [comment]
Ok it sounds like it's working for people so I am going to go ahead and close this issue. Hopefully this mod won't break as badly when the next patch to the game is released (it should at least be a little quicker to fix now)!
